scripts/common/rook.sh

```diff
@@ -87,38 +84,60 @@ function remove_rook_ceph() {
     fi
 
     # remove all rook-ceph CR objects
-    printf "Removing rook-ceph custom resource objects - this may take some time:\n"
-    kubectl delete cephcluster -n rook-ceph rook-ceph # deleting this first frees up resources
-    kubectl get crd | grep 'ceph.rook.io' | awk '{ print $1 }' | xargs -I'{}' kubectl -n rook-ceph delete '{}' --all
-    kubectl delete volumes.rook.io --all
+    log "Removing rook-ceph custom resource objects - this may take some time:\n"
+    if ! kubectl delete cephcluster -n rook-ceph rook-ceph --timeout=60s; then
+        logFail "Unable to delete the rook-ceph CephCluster resource"
+        return 1
+    fi
 
-    # wait for rook-ceph-osd pods to disappear
-    echo "Waiting for rook-ceph OSD pods to be removed"
-    spinner_until 120 rook_ceph_osd_pods_gone
+    log "Removing rook-ceph custom resources"
+    if ! kubectl get crd | grep 'ceph.rook.io' | awk '{ print $1 }' | xargs -I'{}' kubectl -n rook-ceph delete '{}' --all --timeout=60s; then
+        logWarn "Unable to delete the rook-ceph custom resources"
+    fi
 
-    # delete rook-ceph CRDs
-    printf "Removing rook-ceph custom resources:\n"
-    kubectl get crd | grep 'ceph.rook.io' | awk '{ print $1 }' | xargs -I'{}' kubectl delete crd '{}'
-    kubectl delete --ignore-not-found crd volumes.rook.io
+    log "Removing rook-ceph Volume resources"
+    if ! kubectl delete volumes.rook.io --all --timeout=60s; then
+        logWarn "Unable to delete rook-ceph Volume resources"
+    fi
 
-    # delete rook-ceph ns
-    kubectl delete ns rook-ceph
+    log "Removing rook-ceph OSD pods"
+    if ! spinner_until 120 rook_ceph_osd_pods_gone; then
+        logWarn "Unable to delete rook-ceph OSD pods"
```

Comment:
```suggestion
        logWarn "rook-ceph OSD pods were not deleted"
```

(we're not deleting them ourselves, they should disappear because of the cephcluster resource going away)